templates/machinetracker/ip_tracker.html

```diff
@@ -64,6 +67,9 @@
                     {{ row.end_time|date:"Y-m-d H:i:s" }}
                 {% endif %}
             </td>
+            {% if form_data.origin %}
+            <td>{{row.netbox}}</td>
```

Comment:
Actually, it would be better to utilize `row.sysname` here, which is a copy of the router's sysname at the time the record was created. This way, if the netbox is deleted, old log rows will still make sense to the end user.